pkg/multiTenancyPlugins/naming/volumes.go

```diff
@@ -0,0 +1,62 @@
+// volumes
+package namescoping
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/samalba/dockerclient"
+	"errors"
+	"strings"
+	"bytes"
+	"encoding/json"
+	apitypes "github.com/docker/engine-api/types"
```

Comment:
Same comment as above.  "github.com/samalba/dockerclient" is used here because it is used in create container.  I think the change to create container should be handled globally in a separate pull request.